packages/vue-searchbox/src/components/SearchBox.jsx

```diff
@@ -526,6 +541,13 @@ const SearchBox = {
 			event.stopPropagation();
 			event.preventDefault();
 		},
+		withTriggerQuery(eventName, event) {
+			const eventNameInKebabCase = convertToKebabCase(eventName);
```

Comment:
@anik-ghosh-au7 You don't need this utility. We can just emit the events name in kebab case only.